### PR TITLE
Bumps digitalmarketplace-govuk-frontend to 0.3.1

### PR DESCRIPTION
**Pull request by aliuk2012**
Ticket: https://trello.com/c/5aqSYf2P

### CODE REVIEW

**Comment by katstevens** [comment]
Looks like the expected error template isn't being found on these failing tests (as per the other PR).

**Comment by katstevens** [comment]
This is what I see for a 404 on supplier FE: 

supplier-404

File: config.py [hunk 1]
```diff
@@ -88,9 +88,12 @@ class Config(object):
     @staticmethod
     def init_app(app):
         repo_root = os.path.abspath(os.path.dirname(__file__))
+        digitalmarketplace_govuk_frontend = os.path.join(repo_root, "node_modules", "digitalmarketplace-govuk-frontend")
         template_folders = [
             os.path.join(repo_root, "app", "templates"),
-            os.path.join(repo_root, "node_modules", "digitalmarketplace-govuk-frontend", "govuk-frontend"),
+            os.path.join(digitalmarketplace_govuk_frontend),
+            os.path.join(digitalmarketplace_govuk_frontend, "digitalmarketplace", "templates"),
+            os.path.join(digitalmarketplace_govuk_frontend, "govuk-frontend"),
```
Comment (katstevens) [inline config.py]: It's this line that is blocking the error templates from loading - it works fine on the Admin FE though? 🤔

**Comment by lfdebrux** [comment]
We can't update the error templates until the breadcrumbs work has been completed. Going to close this PR for now.

**Pull request closed without merging**